docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.html

```diff
@@ -1,5 +1,18 @@
-<p *ngIf="iconSetClass === 'hpe-icon'">To use an icon simply apply the <code>{{ iconSetClass }}</code>
-    class along with one of the following classes:</p>
+<div *ngIf="iconSetClass === 'hpe-icon'" class="row m-b-md hpe-icon-instructions">
+
+    <div class="col-lg-12 col-md-12 col-sm-12">
```

Comment:
I don't think these divs are needed. If you want to group the elements together to avoid duplicating `ngIf`s, I suggest the `section` element, or maybe `ng-content`.